### PR TITLE
Tour java improvements

### PR DESCRIPTION
**Pull request by gvdongen**
- Incorporated Till's feedback (also fixing pre-release workflow)
- Cleaned up code Java snippets (mainly the calls)
- Improved the JavaTour text
- Added state introspection via psql instead of printing

### CODE REVIEW

File: docs/tour.mdx [hunk 1]
```diff
@@ -357,20 +435,31 @@ The functions of a keyed router have three optional parameters
 
 You can choose the parameter names differently.
 
+For example, the `UserSession` service has the following keyed router:
+
+```typescript
+export const userSessionRouter = restate.keyedRouter({ addTicket, expireTicket, checkout });
+```
+
 As mentioned, for keyed services there can be at most one concurrent invocation at the service level.
```
Comment (pcholakov) [inline docs/tour.mdx:444]:
```suggestion
As mentioned, for keyed services there can be at most one concurrent invocation to any method of the service.
```

File: docs/tour.mdx [hunk 2]
```diff
@@ -357,20 +435,31 @@ The functions of a keyed router have three optional parameters
...
-So this means that if there is a call to `hello()`, then a call to `callMe()`, then `callMe()` will wait in line until `hello()` has finished, if and only if both were invoked with the same value of `key`.
-until the call to `hello()` has finished successfully.
-This also counts for the `maybe()` and `withSomething()` functions,
-even though the key parameter was omitted from their signatures.
+So this means that if there is a call to `addTicket()`, then a call to `expireTicket()`, then `checkout()` will wait in line until `addTicket()` has finished, if and only if both were invoked with the same value of `key`.
+This also counts if you omit the key parameter from the function signature, as we did for the `maybe()` and `withSomething()` functions above.
```
Comment (pcholakov) [inline docs/tour.mdx:446]: This is quite long and maybe hard to follow. Maybe a suggestion:

```suggestion
This means that for the same value of `key`, an unfinished call to `addTicket()` will block subsequent calls to `expireTicket()` or `checkout()` until `addTicket()` has finished. This holds true even if you omit the `key` parameter from the function signature, as we have done with the `maybe()` and `withSomething()` functions above.
```

File: docs/tour.mdx [hunk 3]
```diff
@@ -784,41 +881,41 @@ In the service logs, you see that the `addTicket` function doesn't wait for a re
...
+    var ticketClnt = TicketServiceRestate.newClient(ctx);
+    //highlight-next-line
+    ticketClnt.oneWay().reserve(Ticket.newBuilder().setTicketId(request.getTicketId()).build());
+
+    return BoolValue.of(true);
 }
 ```
 
-Note that `oneWayCall` isn't an asynchronous operation in the Restate SDK, so there is no need to call `await()` on it.
+Note that `oneWay()` isn't an asynchronous operation in the Restate SDK, so there is no need to call `await()` on it.
```
Comment (pcholakov) [inline docs/tour.mdx:897]: I think this reads a tiny bit better:

```suggestion
Note that `oneWay()` is not an asynchronous operation in the Restate SDK, so there is no need to call `await()` on it.
```

**Review by pcholakov — APPROVED** [review]
Some minor suggestions but overall looking great! 👏

**Comment by pcholakov** [comment]
I recently discovered the [curl `--json` flag](https://everything.curl.dev/http/post/json) - it means we can shorten example requests to something like:

```
curl http://localhost:8080/example.UserSession/ExpireTicket --json '{"user_id": "123", "ticket_id":"456"}'
```

Up to you if you want to replace all the occurrences; either way it's a copy-paste operation for the reader.

**Comment by slinkydeveloper** [comment]
@pcholakov I remember there was some concern with `--json`: https://github.com/restatedev/documentation/issues/31
